client/components/help-center/help-center-app.tsx

```diff
@@ -0,0 +1,31 @@
+import config from '@automattic/calypso-config';
```

Comment:
It would be great if we add some front matter explaining the point of this file.